src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -324,6 +324,7 @@ export default class ArrayField extends React.Component {
       : null;
     const isReviewMode = uiOptions.reviewMode;
     const hasTitleOrDescription = (!!title && !hideTitle) || !!description;
+    const uiItemName = (uiOptions.itemName || 'item').toLowerCase();
```

Comment:
made reusable `uiItemName` to distinguish between this and the aria item name (which is per array item whereas this one applies to all of them)